### PR TITLE
Added option to specify path to phantomas executable.

### PR DESCRIPTION
**Pull request by harnash**
Fixed some issues with running test on OSX and added some verbose
error messages when invoking phantomas process.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -33,6 +33,7 @@ Module's API
     
     results = Phantomas(
         url="http://example.com",
+        exec_path="/my/path/to/phantomas"     # optional
```
Comment (macbre) [inline README.rst]: Comma missing

File: phantomas/test/test_phantomas_run.py [hunk 2]
```diff
@@ -2,6 +2,7 @@
 Set of unit tests for Phantomas runner
 """
 import unittest
+import os
```
Comment (macbre) [inline phantomas/test/test_phantomas_run.py]: Let's keep this list in alphabetical order

**Comment by macbre** [comment]
LGTM, thanks for the patch!

**Comment by harnash** [comment]
Should be OK now.

**Comment by macbre** [comment]
:+1:

**Pull request merged**